src/components/v5/frame/ColonyHome/partials/DashboardHeader/partials/ColonyLinks/useMuteColonyItem.tsx

```diff
@@ -62,9 +65,18 @@ export const useMuteColonyItem = (): DropdownMenuItem => {
       onCompleted: async () => {
         await updateUser(user.walletAddress, true);
         setIsMuteToggling(false);
+
+        toast.success(
+          <Toast
+            type="success"
+            title={{ id: 'advancedSettings.toast.changesSaved' }}
+            description={formatText(MSG.toastNotificationsUnmuted)}
+          />,
+        );
```

Comment:
Maybe we could extract this into a separate function?

```
const showSuccessToast = (message: MessageDescriptor) => {
    toast.success(
          <Toast
            type="success"
            title={{ id: 'advancedSettings.toast.changesSaved' }}
            description={formatText(message)}
          />,
        );
}
```

and the call it here `showSuccessToast(MSG.toastNotificationsUnmuted)`